### ISSUE TITLE
[🐛 Bug]: Could not start a new session. Response code 500. Message: Failed to read marionette port

### ISSUE THREAD

**Issue by diemol** [SeleniumHQ/docker-selenium#2020]
### What happened?

When using any of the Firefox images and a random user ID (not the `seluser` ID in Ubuntu), session creation fails and GeckoDriver responds with "Failed to read marionette port".

### Command used to start Selenium Grid with Docker

```shell
docker run --user 24397 -d -p 4444:4444 --shm-size="2g" selenium/standalone-firefox:4.15.0-20231122
```

### Relevant log output

```shell
Could not start a new session. Response code 500. Message: Failed to read marionette port
```

### Operating System

Any of them

### Docker Selenium version (tag)

4.15.0-20231122

**Comment by VietND96** [comment]
This issue is fixed together with PR #2018. Test NodeFirefox images with random user (true) is added back

**Comment by diemol** [comment]
Thank you!
